src/onegov/landsgemeinde/cli.py

```diff
@@ -0,0 +1,107 @@
+""" Provides commands used to initialize org websites. """
+import click
+
+from onegov.core.cli import command_group, pass_group_context, abort
+from onegov.core import html
+from onegov.org.models import Organisation
+from onegov.landsgemeinde.collections import VotumCollection
+from onegov.landsgemeinde.collections import AgendaItemCollection
+from onegov.landsgemeinde.models import AgendaItem
+
+cli = command_group()
+
+
+@cli.command(context_settings={'creates_path': True})
+@click.argument('name')
+@click.option(
+    '--locale',
+    default='de_CH',
+    type=click.Choice(['de_CH', 'fr_CH', 'it_CH'])
+)
+@pass_group_context
+def add(group_context, name, locale):
+    """ Adds an org with the given name to the database. For example:
+
+        onegov-org --select '/onegov_org/evilcorp' add "Evilcorp"
+
+    """
+
+    def add_org(request, app):
+
+        if app.session().query(Organisation).first():
+            abort("{} already contains an organisation".format(
+                group_context.selector))
+
+        app.settings.org.create_new_organisation(app, name, locale=locale)
+
+        click.echo("{} was created successfully".format(name))
+
+    return add_org
+
+
+@cli.command('fill-person-info')
+@pass_group_context
+@click.option('--dry-run', is_flag=True, default=False)
+def fill_person_info(group_context, dry_run=False):
+    """ Takes the first two rows of the votum text and fills them into the
+        name and function field of the person.
+
+    Example:
+    onegov-landsgemeinde --select '/onegov_landsgemeinde/gl' fill-person-info
+    """
```

Comment:
Extend the commit message with a `HINT:` so people know to execute this command after rollout.